gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/transform/SnapshotAggregatorClassVisitor.kt

```diff
@@ -21,7 +23,11 @@ class SnapshotAggregatorClassVisitor(
     desc: String,
     signature: String?,
     exceptions: Array<out String>?
-  ): MethodVisitor {
+  ): MethodVisitor? {
+    if (!includePrivatePreviews && AccessFlags.PRIVATE.isSet(access)) {
```

Comment:
This works but visitMethod is a asm library and AccessFlags is a method in dexlib. They happen to be the same value here but I think it isn’t good to mix libraries like this.
The asm version is this:
```suggestion
    if (!includePrivatePreviews && (access and Opcodes.ACC_PRIVATE == 0)) {) {
```